### PR TITLE
[#174489282] Cfd 13.19

### PR DESCRIPTION
**Pull request by tlwr**
[Story](https://www.pivotaltracker.com/story/show/174489282)

What
----

Upgrade to cf-deployment 13.19, these changes include:

- Remove override for routing-release, we can use upstream
- Remove override for cflinuxfs3, we can use upstream
- Remove override for loggregator agent, we can use upstream
- Bump stemcells to match 621.84, to match upstream
- Use cf-cli version 7, to match upstream
- Twiddle with app-autoscaler acceptance tests, use a git sha from the `develop` branch so that we can deploy an official release, but test using a version of the acceptance tests compatible with v7 for the cf-cli
- Upgrade our custom acceptance tests to use v7 of the cf-cli
- Mark our custom python buildpack test as pending, whilst there is an upstream bug with CAPI that does not infer start command when `cf start`ing an app using v7 of the cf-cli
- Remove a flakey assertion in our common acceptance tests

⚠️  CI is failing because we need to push https://github.com/alphagov/paas-uaa-release/pull/23 to `gds_master` ⚠️ 
Please review anyway

How to review
-------------

Code review, commit by commit

Deploy to your development environment

Who can review
--------------

Not @tlwr

### CODE REVIEW

File: platform-tests/go.mod [hunk 1]
```diff
@@ -10,7 +10,9 @@ require (
 	github.com/bmizerany/pat v0.0.0-20160217103242-c068ca2f0aac // indirect
 	github.com/cloudfoundry-community/go-cfclient v0.0.0-20180906143114-149e140bbeec
 	github.com/cloudfoundry-incubator/cf-test-helpers v1.0.1-0.20190515214722-ff434af77d37
```
Comment (46bit) [inline platform-tests/go.mod:12]: A second version of `github.com/cloudfoundry-incubator/cf-test-helpers` is added in this commit. No idea if that matters but thought it's worth highlighting.

File: concourse/pipelines/create-cloudfoundry.yml [hunk 2]
```diff
@@ -3599,6 +3613,10 @@ jobs:
           - get: cf-manifest
             passed: ['post-deploy']
 
+            # FIXME: when there is a release that supports CF CLI v7 we can use
+            # the submodule in this repo again
+          - get: app-autoscaler-release-git
```
Comment (46bit) [inline concourse/pipelines/create-cloudfoundry.yml:3618]: I'm a bit confused how things work by just `get`ting this. Is undoing it simply a matter of not doing the `get` or am I missing something?

Great work either way

Comment (46bit) [reply concourse/pipelines/create-cloudfoundry.yml:3618]: Oh, https://github.com/alphagov/paas-cf/pull/2444/commits/7b62da657a413204f8aff375dbdb4e77c750fcb3#diff-7f47ee629d360efc30a60c7fc2affb52R59 explained

File: platform-tests/go.mod (hunk 1 above)
Comment (tlwr) [reply platform-tests/go.mod:12]: This might have been during a rebase

**Comment by 46bit** [comment]
I'll try upgrading my dev env to this today. All being well we can merge it first thing Monday.

**Comment by barsutka** [comment]
I deployed this branch to my dev environment, pipeline is mostly green, the only failing job is `custom-broker-acceptance-tests` job due to SQS broker test failures - I believe it's not a showstopper for this PR.
All in all, looks good to me.

**Review by 46bit — APPROVED** [review]

**Comment by tlwr** [comment]
Updated the UAA release to a proper version, just rerunning my dev pipeline then I will merge this

**Pull request merged**